app/src/end2endTest/resources/test-mas/claim-350-7101-outofscope.json

```diff
@@ -16,7 +16,7 @@
     "claimSubmissionDateTime": "2018-11-04T17:45:59Z",
     "claimSubmissionSource": "VA.GOV",
     "conditions": {
-      "name": "string",
+      "name": "blood pressure",
```

Comment:
Made this *anything* but the java class name to ensure that it really does populate the database as expected.